src/index.js

```diff
@@ -818,6 +824,69 @@ async function jsPDFInvoiceTemplate(props) {
     }
   }
 
+  // Define the box parameters
```

Comment:
Add a condition to check if there is any conversion rate related data. Then show it.